src/cli.js

```diff
@@ -128,18 +139,11 @@ if (cli.flags.quiet) {
 }
 
 if (cli.flags.syntax) {
-  optionsBase.syntax = cli.flags.syntax
+  optionsBase.syntax = resolveModule(cli.flags.syntax)
```

Comment:
Won't we have a problem here if `cli.flags.syntax` is one of the standard syntaxes, like `"scss"`?